<!---->
<launch>
  <arg name="port" value="/dev/ttyUSB0"/>

  <node name="uwb_start_YCHIOT" pkg="uwb" type="uwb_start_YCHIOT" output="screen">
    <param name="port" value="$(arg port)"/>
  </node>
  <node name="uwb_calibration" pkg="uwb" type="uwb_calibration" output="screen"/>
</launch>